### ISSUE TITLE
I'm tired of guessing.

### ISSUE THREAD

**Issue by Ebanytiy** [MehVahdJukaar/polytone#11]
despite the fact that the wiki does not mention how to make changes to the player inventory, I managed to do it. but I encountered another problem: the inability to change the position of the slot in the suvival player inventory. The slot position changes only in creative inventory.
![image](https://github.com/MehVahdJukaar/polytone/assets/87265332/9b206df9-4335-4821-ac2a-f51af0de85c5)
![image](https://github.com/MehVahdJukaar/polytone/assets/87265332/79c7bc27-cec8-4753-a357-badc11bdb08e)

**Comment by MehVahdJukaar** [comment]
So player survival screen is a super weird case asts the only one with slots but that doesnt have a menu like all others. You have to use menu_id and "inventory". I added that to the list of menu ids. I guess i need to add a small cheat sheet with the most common targets and their strategies

**Comment by lxly9** [comment]
You will need to use "net.minecraft.class_1723" to target the survival inventory. I will add this class as a reference at the bottom of the wiki with the other container names.

```{
  "target_type": "menu_class",
    "target": "net.minecraft.class_1723",
    "slot_modifiers": [
    {
      "slots": 0,
      "x_offset": -20,
      "y_offset": 22
    },
    {
      "slots": "1->4",
      "x_offset": 27,
      "y_offset": -11
    },
    {
      "slots": "5->8",
      "x_offset": 36,
      "y_offset": -1
    },
    {
      "slots": 45,
      "x_offset": -60,
      "y_offset": -11
    }
  ],
  "special_offsets": {
    "player": {
      "x_offset": 35,
      "y_offset": 0
    }
  },
  "sprites": [
    {
      "x": -102,
      "y": -95,
      "z": -500,
      "width": 204,
      "height": 190,
      "texture": "textures/gui/container/inventory_frame.png"
    }
  ],
  "title_y_offset": -1400
}```
this is a working example

**Comment by lxly9** [comment]
if youre not sure which class to use, use this website to look them up: https://linkie.shedaniel.me/mappings

**Comment by lxly9** [comment]
its also in the wiki ho to modify the inventory (at least how it works on forge)

**Comment by Ebanytiy** [comment]
for me it doesn't work. the position of the slots has not changed.
![image](https://github.com/MehVahdJukaar/polytone/assets/87265332/6ba824c2-2ffc-4146-b20e-4f2c2819a3b7)
![image](https://github.com/MehVahdJukaar/polytone/assets/87265332/fe029c26-f65a-4aa3-adf8-23b682a77a39)

**Comment by lxly9** [comment]
please restart your game after changing inventory slots.

**Comment by Ebanytiy** [comment]
thanks, it works now :)

**Comment by MehVahdJukaar** [comment]
Hm so reloading the game doesn't work? Only works on first pack load? That's not good

**Comment by lxly9** [comment]
yeah, thats what always happened since i wrote the wiki for slotify

**Comment by MehVahdJukaar** [comment]
odd i just tried it and it did change when i reloaded the resoruce pack

**Comment by lxly9** [comment]
fabric or forge?

**Comment by cwmay98** [comment]
I know this has been closed, but I cannot seem to modify the survival inventory properly on Fabric 1.21.1.  I also tried Slotify in 1.20.1 with no changes to the inventory either

My file structure is assets/inventorylayout/polytone/gui_modifiers/inventory_layout_modifier.json

My json is written:
```
{  
  "target_type": "menu_class",  
  "target": "net.minecraft.class_1723",  
  "slot_modifiers": [
	{  
	"slots": 45,
	"x_offset": 3,
	"y_offset": 0
	},
	{
	"slots": "1->4",
	"x_offset": -90,
	"y_offset": 0
	},
	{
	"slots": 0,
	"x_offset": -92,
	"y_offset": 1
	},
	{
	"slots": "5->8",
	"x_offset": 140,
	"y_offset": 0
	}
	], 
	
    "special_offsets": {
    "player": {
      "x_offset": 100,
      "y_offset": 0
    }
  }***
}
```
I tried both InventoryMenu and net.minecraft.class_1723
I also couldn't find a way to move the recipe book... The wiki calls the item knowledge_book, player data calls it recipeBook, and Linkie has a few options. I am not certain which would be the button in the player inventory

What am I doing wrong here?

Edit:
I missed a curly bracket where it is marked with ***. It is working in 1.21.1, apologies for my confusion. I will try to figure out how to move the recipe book now.
